src/attributes.jl

```diff
@@ -21,9 +33,13 @@ Abstract supertype for attribute objects that can be used to set or get attribut
 """
 abstract type AbstractConstraintAttribute end
 
-const AnyAttribute = Union{AbstractInstanceAttribute, AbstractVariableAttribute, AbstractConstraintAttribute}
+const AnyAttribute = Union{AbstractSolverParameter, AbstractInstanceAttribute, AbstractVariableAttribute, AbstractConstraintAttribute}
```

Comment:
Is a parameter a special kind of attribute or is it a different thing? We should be precise about this.

Reply:
I think it is better to split but still have a common supertype